### ISSUE TITLE
possible to remove python package and cripple conda

### ISSUE THREAD

**Issue by ijstokes** [conda/conda#4751]
There are several paths that can lead to the removal of Python without any replacement (i.e. a different version), thus leading to a borked conda installation.  I think it should be made (almost) impossible to remove Python from the environment where conda relies upon it.

**Comment by kalefranz** [comment]
@mcg1969 I've seen this recently also.  @jbednar showed me output where `conda remove ipywidgets` also triggered a removal of python in the root environment.  Sometime to be watching out for...

**Comment by mcg1969** [comment]
Indeed, I saw Jim's example, and I can't for the life of me figure out why it would do that, nor can I reproduce it. I agree that adding a safety valve to prevent its removal would be worth adding.

**Comment by mcg1969** [comment]
First, what we need is to hasten the arrival of a private environment for conda

**Comment by mcg1969** [comment]
The only thing I can think w.r.t. Jim's case was that his conda env was horribly broken, so the bad_installed path got involved. That function needs to die.

**Comment by jbednar** [comment]
If the envt was broken, this behavior was the only symptom that I've noticed; it's been reliable in daily use otherwise.  That said, after overwriting the installation with a new miniconda (which itself required moving some conflicting directories out of pkgs/) I installed ipywidgets again and then removed it, with no problems.  So indeed something about my previous envt must have been different from the new one.   Who knows; I've been using this same envt for years now, across probably a hundred conda upgrades. :-)
